### PR TITLE
Fix register aa1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the sign up of users is not really stable.

**What is the new behavior (if this is a feature change)?**
- The sign up has been improved and should now be more stable.

### CODE REVIEW

File: src/store/actions/authActions.js [hunk 1]
```diff
@@ -97,35 +97,47 @@ export const signUp = (newUser) => {
           firebase
             .auth()
             .createUserWithEmailAndPassword(newUser.email, newUser.password)
-            .then((response) => {
+            .then(async (response) => {
```
Comment (Theralun) [inline src/store/actions/authActions.js:100]: This does not have to be `async`.
```suggestion
            .then((response) => {
```

File: src/store/actions/authActions.js [hunk 2]
```diff
@@ -97,35 +97,47 @@ export const signUp = (newUser) => {
...
             })
             .then(() => {
+              console.log("Success Signup");
+
               dispatch({
                 type: "SIGNUP_SUCCESS",
               });
             })
             .catch((err) => {
+              console.log(err);
```
Comment (Theralun) [inline src/store/actions/authActions.js:139]: Can remove this `console.log(err)`.

File: src/components/organisms/Posts/index.jsx [hunk 3]
```diff
@@ -185,309 +184,347 @@ class Posts extends React.Component {
...
+  };
+};
 
-export default connect(mapStateToProps,mapDispatchToProps)(Posts);
+export default connect(mapStateToProps, mapDispatchToProps)(Posts);
 
-/** 
+/**
  * SPDX-License-Identifier: (EUPL-1.2)
  * Copyright © 2019 Christian Aichner
```
Comment (LucaAllmaier) [inline src/components/organisms/Posts/index.jsx:529]: Update license

File: src/components/organisms/RegisterForm/index.jsx [hunk 4]
```diff
@@ -481,490 +462,522 @@ class HomePage extends React.Component {
...
+                            containerClass="my-2"
+                            checked={this.state.checkPrivacy}
+                            onChange={this.handleCheckboxChange}
+                            name="checkPrivacy"
+                            id="checkbox30"
+                            required
+                          />
+                          <MDBInput
+                            label="I agree, that the data from this form is being stored and used by 
+                  SithCult in order for you to receive any service"
```
Comment (LucaAllmaier) [inline src/components/organisms/RegisterForm/index.jsx:922]: Shouldn't it be "in order for me to receive any service"?

File: src/components/organisms/RegisterForm/index.jsx [hunk 5]
```diff
@@ -976,19 +989,22 @@ const mapStateToProps = (state) => {
...
-export default connect(mapStateToProps,mapDispatchToProps)(withRouter(HomePage));
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(withRouter(HomePage));
 
-/** 
+/**
  * SPDX-License-Identifier: (EUPL-1.2)
  * Copyright © 2019 Christian Aichner
```
Comment (LucaAllmaier) [inline src/components/organisms/RegisterForm/index.jsx:1009]: Update license

**Pull request merged**